### PR TITLE
Mastery Exercise

### PR DESCRIPTION
**Pull request by Anulfo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bower_components
+bower.json
```
Comment (gregkorte) [inline .gitignore]: This file should be in your remote repo. It allows me to pull down the files required to run your app.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            <div class="container">
+                <h1>Luxury Used Cars</h1>
+            </div>
+        </header>
+            <div class="navbar navbar-light bg-faded">
+                <input type="text" name="carSpecToChange" id="inputField" class="form-text">
+                <button id="doneButton" class="btn btn-success">DONE</button>
+            </div>
+        <div id="carCards" class="container">
+            <div class="row">
```
Comment (gregkorte) [inline index.html]: This row should not be hard coded but rather dynamically created by javascript. What would happen if you added more cars? Would each set of three end up in their own rows?

File: .gitignore (hunk 1 above)
Comment (Anulfo) [inline .gitignore]: I corrected, well I think over corrected.  I misunderstood the mail and push the bower_components in too. Now i know is just the bower.json, so you can now wich dependencies i used? Did I get it now?

**Comment by gregkorte** [comment]
Looks good, Anulfo! You are good to merge. :+1:

**Pull request merged**